src/main.js

```diff
@@ -450,6 +464,32 @@ function validateConfig(configParams) {
   }
 }
 
+/**
+ * converts a timezone string into the minute repräsentation of it
+ * @param {string} timezone 
+ * @returns {number | null} timezone in minutes or null if invalid
+ */
+ function convertTimezoneToMinutesOffset(timezone) {
```

Comment:
Check out https://github.com/bitmovin-engineering/dashboard/blob/develop/src/utils/generic/DateUtils.ts#L217.
This is how we do it in the dashboard and covers more cases.

Reply:
Thanks for the reference i tried this code but beside the issues i had with converting this to plain javascript it didn't worked out as well as the code before